docs/index.md

```diff
@@ -0,0 +1,23 @@
+# Beetbox - Drupal development on Vagrant [![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox)
+
+## Overview
+
+Beetbox is a Vagrant configuration that provides your team with a versatile development environment for Drupal projects using Virtualbox for VMs (virtual machines).
```

Comment:
Are we sticking with 'environment for Drupal projects'? I though beetbox can be used for any LAMP/LEMP stack. I think this is a bit misleading.